test/blackbox/common/mock/BlackboxMockConsumer.h

```diff
@@ -42,6 +42,12 @@ class BlackboxMockConsumer : public LogConsumer
         return mEntriesConsumed;
     }
 
+    void ClearEntries()
```

Comment:
As commented before, I don't know if this new method is necessary. I don't mind having it for the future, but then it may be nice to follow naming style rules: i.e. `clear_entries`